build.gradle

```diff
@@ -3,26 +3,22 @@ plugins {
 	id 'eclipse'
 	id 'idea'
 	id 'maven-publish'
-	id 'fabric-loom' version '0.6-SNAPSHOT'
-	id 'net.minecrell.licenser' version '0.4.1'
+	id 'fabric-loom' version "0.7-SNAPSHOT"
+	id 'net.minecrell.licenser' version "0.4.1"
 	id "org.ajoberstar.grgit" version "3.1.1"
 	id 'com.matthewprenger.cursegradle' version "1.4.0"
-	id 'maven'
 	id 'signing'
 }
-def ENV = System.getenv()
 
-class Globals {
-	static def baseVersion = "1.1.0"
-	static def mcVersion = "1.8.9"
-	static def yarnVersion = "+build.202103291533"
-	static def testmodsProject = "testmods"
-}
+def ENV = System.getenv()
 
-version = Globals.baseVersion + "+" + Globals.mcVersion
+setVersion("${project.base_version}+${project.mc_version}")
 logger.lifecycle("Building Fabric: " + version)
 
-import org.apache.commons.codec.digest.DigestUtils
+
+import net.fabricmc.loom.task.RemapJarTask
+import org.apache.commons.codec.digest.
```

Comment:
This doesn't seem right

Reply:
Well it is. There was an unnecessary qualified reference.